apps/coordinator/src/components/ScriptExplorer/SignatureImporter.jsx

```diff
@@ -398,21 +391,8 @@ class SignatureImporter extends React.Component {
 
         let publicKey;
         try {
-          const sequence = enableRBF ? 0xfffffffd : 0xffffffff;
-          const args = {
-            network,
-            inputs: inputs.map((input) => {
-              const convertedInput = convertLegacyInput(input);
-              return {
-                ...convertedInput,
-                sequence: sequence, // Apply the same RBF sequence as in finalizeOutputs so if RBF signalling we don't lose that
-              };
-            }),
-            outputs: outputs.map(convertLegacyOutput),
-          };
-          const psbt = getUnsignedMultisigPsbtV0(args);
           publicKey = validateMultisigPsbtSignature(
-            psbt.toBase64(),
+            unsignedPSBT,
```

Comment:
rather than recreate the psbt based on the internal state, we can use the unsigned psbt we have already built